adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/JaxRsInterfaceGenerator.java

```diff
@@ -73,8 +72,10 @@
     private TypeSpec createInterfaceFor(final Resource resource) {
         final TypeSpec.Builder interfaceSpecBuilder = interfaceSpecFor(resource);
 
-        resource.getActions().values().forEach(action ->
-                interfaceSpecBuilder.addMethods(forEach(action)));
+        resource.getActions().values().forEach(action -> {
```

Comment:
Could we use a filter to remove those that are not valid rather than encapsulating the check within?